lib/Site.js

```diff
@@ -260,13 +263,16 @@ Site.prototype.generate = function (baseUrl) {
 };
 
 /**
- * Build all pages of the site
+ * Build all pages and the index of the site
```

Comment:
But I see this in `Site.prototype.generate`:

```diff
+ .then(() => this.buildIndex())
  .then(() => this.buildSourceFiles())
```

How about:

```js
.then(() => this.collectAddressablePaths()) // The current buildIndex
.then(() => this.buildSourceFiles())
.then(() => this.writeSiteData())
```

Let's stick to one term, rather than mixing with "Site index".
"siteData" is consistent with "pageData" in `Site.prototype.createPageData` 👍